app/models/friend_request.rb

```diff
@@ -0,0 +1,30 @@
+class FriendRequest < ApplicationRecord
```

Comment:
FriendRequest works well in the console! :) 

Great job on the FriendRequest model!